src/lib/cdt-proxy.ts

```diff
@@ -23,6 +23,9 @@ import { Breakpoint } from './breakpoint';
 import { onHttpRequest } from './cdt-proxy-http';
 import { JerryMessageScriptParsed } from './protocol-handler';
 
+export type PausedReason = 'exception' | 'debugCommand' | 'other';
```

Comment:
the first | looks bolded, is it a format error of github or normal?

Reply:
I do see that too... I think it's a font/resolution issue on our monitors. If I zoom in one click, the other one gets bolded. :)

Thanks for taking a look.